### PR TITLE
Add Vortex Xspress3 Class

### PR DESCRIPTION
**Pull request by lucianocguedes**
Add Vortex Xspress3 Class

### CODE REVIEW

File: py4syn/epics/VortexXspress3Class.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+        self.pvStatus = PV(pv + ':Acquire_RBV', callback=self.onAcquireChange)
+        self.pvAcquireTime = PV(pv + ':AcquireTime')
+        self.pvClear = PV(pv + ':ERASE')
+
+        self.pvMcaCounters = []
+        self.pvStatusScan = PV(pv + ':Acquire_RBV.SCAN')
+        self.pvStatusScan.put(9)
+
+        self.inicio = time()
+        self.fim = 0
```
Comment (hhslepicka) [inline py4syn/epics/VortexXspress3Class.py:60]: Change `inicio` and `fim` to english, maybe?

File: py4syn/epics/VortexXspress3Class.py [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+        # Channels 1-4
+        for i in range(1, 5):
+            for j in range(1, 5):            
+               self.pvMcaCounters.append(PV(pv + ':C' + str(j) + '_ROI'+str(i)+':Value_RBV'))
```
Comment (hhslepicka) [inline py4syn/epics/VortexXspress3Class.py:65]: Suggestion:
```python
self.pvMcaCounters.append(PV('{}:C{}_ROI{}:Value_RBV'.format(pv, j, i))
```

File: py4syn/epics/VortexXspress3Class.py [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+        Returns false indicating that vortex cannot be used as a counter monitor.
+        """
+        return False
+
+
+    def canStopCount(self):
+        """
+        Returns true indicating that vortex has a stop command.
+        """
+        return True
```
Comment (hhslepicka) [inline py4syn/epics/VortexXspress3Class.py:145]: If the vortex has a `stop` command you should properly implement also the `stopCount` or `close`

**Review by hhslepicka — COMMENTED** [review]
Just a couple of comments. Feel free to take into consideration or not. 👍

**Comment by lucianocguedes** [comment]
> Just a couple of comments. Feel free to take into consideration or not.

Thanks @hhslepicka for you comments. In my revision, I took then into consideration and remove some unnecessary imports. Thanks again!

**Comment by hhslepicka** [comment]
@lucianocguedes awesome! 
By the way, you don't need to close a Pull Request if changes are requested... you can just make the changes using the same branch and keep pushing to it... the changes will show up here and maintainers can keep better track of what was done! 👍 
But if the workflow defined by you guys is to close the PR and open a new one all is good as well! 😃

**Pull request closed without merging**